server/server.go

```diff
@@ -925,7 +925,7 @@ func (s *Server) loadConfig(configurations types.Configurations, globalConfigura
 						redirectHandlers[entryPointName] = handlerToUse
 					}
 				}
-				if backends[entryPointName+providerName+frontend.Backend] == nil {
+				if backends[entryPointName+providerName+frontend.Hash()] == nil {
```

Comment:
Could this string be stored, and referenced, instead of being recalculated multiple times?

Reply:
Yes, that should be doable. I'll just need to make sure the frontend isn't changed between usages of a store string.

Reply:
I was just thinking inside the loop. Instead of calculating `.Hash()` three times.
`entryPointName+providerName+frontend.Hash()` in your PR.

Reply:
I hadn't checked if shady things where done to the frontend in the loop (although that would also have been a bug if it did).

Implemented in b716cb2b3c892a678d9cac77954dea11ae9439bc.